src/test/rustdoc/intra-doc/extern-crate-only-used-in-link.rs

```diff
@@ -1,19 +0,0 @@
-// This test is just a little cursed.
```

Comment:
Please add this test back, it should pass with and without unconditional loading.